lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -356,7 +356,8 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
     ENV LIVEBOOK_TEAMS_NAME "#{socket.assigns.hub.hub_name}"
     ENV LIVEBOOK_TEAMS_OFFLINE_KEY "#{socket.assigns.hub.org_public_key}"
 
-    CMD [ "/app/bin/livebook", "start" ]\
+    ENV LIVEBOOK_APPS_PATH_WARMUP "manual"
+    RUN /app/bin/warmup_apps.sh\
```

Comment:
We don't need to specify CMD, the one from base image is used unless overridden :)